lib/shared/utils.js

```diff
@@ -22,18 +22,20 @@ const errors = require('./errors');
 /**
  * @summary Minimum percentage value
  * @constant
- * @private
+ * @public
  * @type {Number}
  */
 const PERCENTAGE_MINIMUM = 0;
+exports.PERCENTAGE_MINIMUM = PERCENTAGE_MINIMUM;
```

Comment:
Done! Took a bit of fiddling, but I eventually managed to get `npm test` passing again :)